content/docs/gke/monitoring.md

```diff
@@ -4,162 +4,7 @@ description = "Logging and monitoring for Kubeflow"
-```
-
-#### Filter with labels
-
-The new agents also support querying for logs using pod labels
-For example:
-
-```
-resource.type="k8s_container"
-resource.labels.cluster_name="${CLUSTER}"
-metadata.userLables.${LABEL_KEY}="${LABEL_VALUE}"
-```
-
-# Monitoring
-
-## Stackdriver on GKE
-The new [Stackdriver Kubernetes Monitoring](https://cloud.google.com/monitoring/kubernetes-engine)
-provides single dashboard observability and is compatible with Prometheus data model.
-
-See this [doc](https://cloud.google.com/monitoring/kubernetes-engine/observing) for more
-details on the dashboard.
-
-Stackdriver by default provides container level CPU/memory metrics.
-We can also define custom Prometheus metrics and view them on the Stackdriver dashboard.
-See for more [detail](https://cloud.google.com/monitoring/kubernetes-engine/prometheus).
-
-## Prometheus
-
-### Kubeflow Prometheus component
-Kubeflow provides a Prometheus [component](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/gcp/prometheus.libsonnet).
-To deploy the Prometheus component:
-
-```
-ks generate prometheus prom --projectId=YOUR_PROJECT --clusterName=YOUR_CLUSTER --zone=ZONE
-ks apply YOUR_ENV -c prom
-```
-
-The prometheus server will scrape the services with annotation `prometheus.io/scrape=true`.
-See for more [detail](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/gcp/prometheus.yml#L75) 
-and an [example](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/gcp/metric-collector.libsonnet#L83).
-
-#### Export metrics to Stackdriver
-The Prometheus server will export metrics to Stackdriver, as
-[configured](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/gcp/prometheus.yml#L127).
-We are using an [image](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/gcp/prometheus.libsonnet#L170)
-provided by Stackdriver. See Stackdriver [doc](https://cloud.google.com/monitoring/kubernetes-engine/prometheus)
-for more detail, but you don't need to change anything here.
-
-If you don't want to export metrics to Stackdriver, remove the `remote_write` part in the `prometheus.yml`,
-and use a native Prometheus [image](https://hub.docker.com/r/prom/prometheus/tags/).
-
-### Metric collector component for IAP (GKE only)
-Kubeflow also provides a metric-collector [component](https://github.com/kubeflow/kubeflow/tree/master/metric-collector).
-This component periodically pings your Kubeflow endpoint and provides a
-[metric](https://github.com/kubeflow/kubeflow/blob/master/metric-collector/service-readiness/kubeflow-readiness.py#L21) 
-of whether the endpoint is up or not. To deploy it:
-
-```
-ks generate metric-collector mc --targetUrl=YOUR_KF_ENDPOINT
-ks apply YOUR_ENV -c mc
-```
+To configure and use Prometheus with Kubernetes Engine Monitoring, see [Using Prometheus](https://cloud.google.com/monitoring/kubernetes-engine/prometheus) for details.
```

Comment:
It'd be good to let the reader know they're hopping over to the GCP docs. Suggestion: Change this:

```
see [Using Prometheus](https://cloud.google.com/monitoring/kubernetes-engine/prometheus) for details.
```

to this:

```
see [the GCP documentation](https://cloud.google.com/monitoring/kubernetes-engine/prometheus).
```